seed/ui/src/state.js

```diff
@@ -95,9 +95,10 @@ const filterOnline = peer => {
 const data = {
   seed: {
     name: "seedling.radicle.xyz",
-    address: "hybh5c...7ye83k@seedling.radicle.xyz:12345",
+    address:
+      "hybh5cb7spafgs7skjg6qkssts3uxht31zskpgs4ypdzrnaq7ye83k@seedling.radicle.xyz:12345",
```

Comment:
```suggestion
      "seedling.radicle.xyz:12345",
      "peerId": "hybh5cb7spafgs7skjg6qkssts3uxht31zskpgs4ypdzrnaq7ye83k",
```

This is the shape of the data and it probably needs to go into the infoStore.